### PR TITLE
Fix beta->latest->beta upgrade loop

### LINKED ISSUE #6007: On Alpha channel Lens constantly wants to update between 6.0.0-latest and 6.0.1-beta and back
Screen Shot 2022-08-09 at 11 54 15 AM

If you have the Update Channel set to Alpha Lens will keep reporting update available, and keep updating on exit, back and forth between 6.0.0-latest and 6.0.1-beta

Beta and Stable channels work as expected.

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Sebastian Malton <sebastian@malton.name>

fixes #6007

### CODE REVIEW

File: src/common/vars/release-channel.injectable.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import type { UpdateChannelId } from "../application-update/update-channels";
+import appSemanticVersionInjectable from "./app-semantic-version.injectable";
+
+const currentReleaseIdInjectable = getInjectable({
+  id: "release-channel",
```
Comment (jim-docker) [inline src/common/vars/release-channel.injectable.ts:10]: naming seems a bit loose here. The id should be `"current-release-id"`? and the file name similarly changed?

**Review by jim-docker — COMMENTED** [review]
testing...

File: src/common/vars/release-channel.injectable.ts (hunk 1 above)
Comment (Nokel81) [reply src/common/vars/release-channel.injectable.ts:10]: Good catch

**Comment by jim-docker** [comment]
This may be a pre-existing issue, where there is a 6.0.1-latest, 6.0.2-alpha, and 6.0.2-beta available on S3 (note the Lens binaries are all the same, built (`yarn dist`) from this PR branch, except for the version set in the package.json):

- launched 6.0.0-latest (also same binary), on alpha channel 
- update is available (6.0.2-alpha)
- relaunch to update
- version is now 6.0.2-alpha
- update is available (6.0.2-beta)

The first update should be straight to 6.0.2-beta? Or it shouldn't immediately update to beta after updating to alpha.

Same result starting from a binary built from master (which I called 6.0.1-latest)

**Comment by Nokel81** [comment]
@jim-docker That is a good catch. I think it is a pre-existing issue which should also be fixed.

I think the correct solution would be go straight to `6.0.2-beta` ignoring the alpha.

I don't think that it should be fixed in this PR though.

**Comment by jim-docker** [comment]
> I don't think that it should be fixed in this PR though.

I can make a new issue

**Review by jim-docker — APPROVED** [review]
tested various updates from alpha, beta, stable channels. Can no longer reproduce beta->latest->beta loop bug.

**Comment by jim-docker** [comment]
> I can make a new issue

https://github.com/lensapp/lens/issues/6251

**Review by Iku-turso — APPROVED** [review]

**Pull request merged**